app-emulation/docker/docker-9999.ebuild

```diff
@@ -236,24 +233,42 @@ src_compile() {
 		fi
 	done
 
-	go_export
+	pushd components/engine || die
 
-	# verbose building
-	export BUILDFLAGS="-x -v"
+	if use hardened; then
```

Comment:
Previously, it added `-fno-PIC` based on the value of `gcc-specs-pie` instead of the `hardened` flag.  The `gcc-specs-pie` function evaluates to `true` in our SDK, so I don't think we're in a different position.  Our amd64 profile inherits from Gentoo's hardened profile, so it seems these kinds of options are intended to be used.